src/components/accordion/index.md.njk

```diff
@@ -17,83 +17,119 @@ The accordion component lets users show and hide sections of related content on
 
-## Accordions, tabs and details
+Avoid putting accordions within accordions, as it will make content difficult to find.
 
-Accordions, [tabs](/components/tabs/) and [details](/components/details/) all hide sections of content which a user can choose to reveal.
-
-Consider using an accordion instead of tabs if users might need to view more than one section at a time.
+Do not use the accordion component if the amount of content it would need to contain will make the page slow to load.
 
-You should also take into account the number of sections of content – accordions display vertically, so they can fit more sections than horizontal tabs.
+## Decide between using accordions, tabs and details
 
-Tabs may work better for users who need to switch quickly between 2 sections. Accordions push other sections down the page when they open, but tabs do not move which makes it easier to switch.
+Accordions, [tabs](/components/tabs/) and [details](/components/details/) all work by hiding sections of content which a user can choose to reveal. Be careful when using any of these components within one another.
 
-Consider using an accordion instead of the [details](/components/details/) component if there are multiple related sections of content. The details component might be better if you only have one or two sections of content. The details component is less visually prominent than an accordion, so tends to work better for content which is not as important to users.
+If you decide to use one of these components, consider if:
+- the user needs to look at more than one section at a time — an accordion can show multiple sections at a time, unlike tabs
+- the user needs to switch quickly between sections — tabs can show content without pushing other sections down the page, unlike accordions
+- there are many sections of content — accordions can fit more sections as they’re arranged vertically, unlike tabs which are arranged horizontally
+- there’s only one or two pieces of short, less important content — the details component is more suitable as it’s visually smaller and less prominent than an accordion or tabs
 
 ## How it works
 
 There are 2 ways to use the accordion component. You can use HTML or, if you’re using Nunjucks or the GOV.UK Prototype Kit, you can use the Nunjucks macro.
 
 The accordion component uses JavaScript. When JavaScript is not available, users will see all the content displayed with the section labels as headings.
 
-### Find out how your users interact with the content
+### Section heading buttons
+
+An accordion will usually start with all sections hidden. To show a section, the user can interact anywhere in the heading button.
 
-Users might need the sections they have opened to stay open if they leave and then return to the page. You can configure the accordion component to stay open.
+The heading button includes all of these areas:
+- heading text
+- summary text (if you decide to add one)
```

Comment:
It feels like we switch between 'summary text' and 'summary line' in this guidance a bit - should we be consistent?

Reply:
Good point, thanks Vanita. I'm going to move these all to 'summary line' — calling it text helps tie it with 'button text' but 'line' is probably more common and is better with the "keep it short" message.

Reply:
```suggestion
- summary line (if you decide to add one)
```